constants.gradle

```diff
@@ -8,7 +8,7 @@ project.ext {
     targetSdk = 27
     minSdk = 14
 
-    firebaseVersion = '11.8.0'
+    firebaseVersion = '12.0.0'
```

Comment:
Let's wait for `12.0.1` ... `12.0.0` turned out to be kinda broken by some wayward ProGuard.

Reply:
Yeah, I just saw your other comment. SGTM!

Reply:
@samtstern Can you also unmark the `CredentialsClient` as `@Hide`? That makes lint think we're not allowed to use its methods... pretty sure you guys only meant to mark the constructors as `@Hide` 😄.